examples/SnapML/nyc_taxi_tip_predict_notebook.ipynb

```diff
@@ -1617,7 +1617,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.4"
+   "version": "3.6.10"
```

Comment:
Looks like this notebook went from py37 to py36?
Is that expected?

Reply:
The work here was to re-run this notebook with the latest packages from WML-CE 1.7.0.
Since we support both python 3.6 and 3.7 this is fine.